tests/commands/test_patch.py

```diff
@@ -0,0 +1,63 @@
+"""
+:Description: Tests the `patch` CLI
+"""
+
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from conda_recipe_manager.commands.patch import patch
+from tests.file_loading import TEST_FILES_PATH
+from tests.smoke_testing import assert_cli_usage
+
+
+def test_usage() -> None:
+    """
+    Smoke test that ensures rendering of the help menu
+    """
+    assert_cli_usage(patch)
+
+
```

Comment:
We should add tests to ensure we fail correctly when each file is given a path that doesn't exist.

Reply:
but click does that for us anyways.

Reply:
I suppose this is where the line between `unit` and `integration` testing gets a bit blurry. But it is a pretty easy thing to test. 

Here's a hypothetical that could happen/I've seen before (not specifically with `click` but with other libraries):

There could be some day in the future where we decide not to use `click` any more. In that event, we replace `click`. If there is no test and the new system doesn't work like `click` does, we could have a regression. Best to cover our butts now than have it blow up in our face.